EventsExpress/EventsExpress.csproj

```diff
@@ -1,92 +0,0 @@
-<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
Why was this proj file removed?

Reply:
added this file and changed path for logging by default